*** install_logging.yaml	2018-03-23 11:58:13.000000000 -0400
--- install_logging.yaml.fix	2018-03-23 12:06:05.000000000 -0400
***************
*** 84,90 ****
  
      openshift_logging_elasticsearch_storage_type: "{{ elasticsearch_storage_type }}"
      openshift_logging_elasticsearch_pvc_pv_selector: "{{ openshift_logging_es_pv_selector }}"
!     openshift_logging_elasticsearch_pvc_storage_class_name: "{{ openshift_logging_es_pvc_storage_class_name }}"
      openshift_logging_elasticsearch_nodeselector: "{{ openshift_logging_es_nodeselector if outer_item.0.nodeSelector | default(None) is none else outer_item.0.nodeSelector }}"
      openshift_logging_elasticsearch_storage_group: "{{ [openshift_logging_es_storage_group] if outer_item.0.storageGroups | default([]) | length == 0 else outer_item.0.storageGroups }}"
      _es_containers: "{{ outer_item.0.containers}}"
--- 84,90 ----
  
      openshift_logging_elasticsearch_storage_type: "{{ elasticsearch_storage_type }}"
      openshift_logging_elasticsearch_pvc_pv_selector: "{{ openshift_logging_es_pv_selector }}"
!     openshift_logging_elasticsearch_pvc_storage_class_name: "{{ openshift_logging_es_pvc_storage_class_name  | default() }}"
      openshift_logging_elasticsearch_nodeselector: "{{ openshift_logging_es_nodeselector if outer_item.0.nodeSelector | default(None) is none else outer_item.0.nodeSelector }}"
      openshift_logging_elasticsearch_storage_group: "{{ [openshift_logging_es_storage_group] if outer_item.0.storageGroups | default([]) | length == 0 else outer_item.0.storageGroups }}"
      _es_containers: "{{ outer_item.0.containers}}"
***************
*** 111,117 ****
  
      openshift_logging_elasticsearch_storage_type: "{{ elasticsearch_storage_type }}"
      openshift_logging_elasticsearch_pvc_pv_selector: "{{ openshift_logging_es_pv_selector }}"
!     openshift_logging_elasticsearch_pvc_storage_class_name: "{{ openshift_logging_es_pvc_storage_class_name }}"
  
    with_sequence: count={{ openshift_logging_es_cluster_size | int - openshift_logging_facts.elasticsearch.deploymentconfigs.keys() | count }}
    loop_control:
--- 111,117 ----
  
      openshift_logging_elasticsearch_storage_type: "{{ elasticsearch_storage_type }}"
      openshift_logging_elasticsearch_pvc_pv_selector: "{{ openshift_logging_es_pv_selector }}"
!     openshift_logging_elasticsearch_pvc_storage_class_name: "{{ openshift_logging_es_pvc_storage_class_name  | default() }}"
  
    with_sequence: count={{ openshift_logging_es_cluster_size | int - openshift_logging_facts.elasticsearch.deploymentconfigs.keys() | count }}
    loop_control:
